addons/web_editor/static/src/js/common/utils.js

```diff
@@ -408,6 +408,23 @@ function _getColorClass(el, colorNames, prefix) {
     const prefixedColorNames = _computeColorClasses(colorNames, prefix);
     return el.classList.value.split(' ').filter(cl => prefixedColorNames.includes(cl)).join(' ');
 }
+/**
+ * Checks if the current device is mobile.
+ *
+ * @private
+ * @param {Object} self - context object ("this")
+ * @returns {boolean}
+ */
+function _isMobile(self) {
+    let isMobile;
+    self.trigger_up('service_context_get', {
+        callback: (ctx) => {
+            isMobile = ctx['isMobile'];
+        },
+    });
+
+    return isMobile;
+}
```

Comment:
What do you mean? I replaced every occurrence where I used it.

Reply:
The purpose of a utility function is (among other things) to prevent having duplicated code in several places. Here you have added this new utility, it's great but you should make sure that we use this utility in the whole codebase. So you also have to adapt the `toggleDeviceVisibility()` of `DeviceVisibility` option, the `_updateInvisibleDOM` of the `SnippetsMenu` widget